lib/perl/Genome/Qc/Tool/WithVariationListVcf.pm

```diff
@@ -0,0 +1,32 @@
+package Genome::Qc::Tool::WithVariationListVcf;
+
+use strict;
+use warnings;
+use Genome;
+
+class Genome::Qc::Tool::WithVariationListVcf {
+    is_abstract => 1,
+    has => [
+        variation_list_build_id => {
+            is => 'Text',
+            is_optional => 1,
+        }
+    ],
+};
+
+sub variation_list_vcf_file {
+    my $self = shift;
+    my $variation_list_build = Genome::Model::Build::ImportedVariationList->get($self->variation_list_build_list);
```

Comment:
Oops, that was supposed to be `variation_list_build_id`. I implemented your suggestion.